### PR TITLE
feat: add meeting link in workflow reminder template

### LINKED ISSUE #17922: Redirect to sign up page before accepting scheduled meetings
**Issue Summary**
The meeting link is included in the confirmation email but is missing from reminder emails. This inconsistency creates confusion for users, as they rely on reminders for last-minute access to meeting details.

**Steps to Reproduce**
- Schedule a meeting via cal.com.
- Receive the confirmation email with the meeting link (works as expected).
- Wait for the reminder email sent closer to the meeting time.
- Notice that the reminder email does not include the meeting link.
**Actual Results**
The meeting link is missing from reminder emails, making it difficult for users to join the meeting directly from the reminder.
**Expected Results**
The reminder email should include the meeting link, ensuring users have all necessary details to join the meeting.

**Evidence**
Attached are screenshots of  the reminder email (without link).

Screenshot 2024-11-30 192854

### PR DESCRIPTION
**Pull request by kart1ka**
## What does this PR do?

It includes meeting link in the default reminder template for email and sms reminder workflows.

- Fixes #17922 (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [ ] I confirm automated tests are in place that prove my fix is effective or that my feature works.

### CODE REVIEW

File: packages/features/ee/workflows/lib/reminders/emailReminderManager.ts [hunk 1]
```diff
@@ -213,6 +215,8 @@ export const scheduleEmailReminder = async (args: scheduleEmailReminderArgs) =>
   // Allows debugging generated email content without waiting for sendgrid to send emails
   log.debug(`Sending Email for trigger ${triggerEvent}`, JSON.stringify(emailContent));
 
+  console.log("========================EMAIL CONTENT=======================", emailContent);
+
```
Comment (Praashh) [inline packages/features/ee/workflows/lib/reminders/emailReminderManager.ts:219]: remove logs

Comment (kart1ka) [reply packages/features/ee/workflows/lib/reminders/emailReminderManager.ts]: Done

**Review by Praashh — COMMENTED** [review]
Screenshot 2024-12-02 at 3 39 38 PM

we already have the meeting URL in the reminder template, users can add the meeting URL, cancel the URL, etc if they want.

**Comment by kart1ka** [comment]
> Screenshot 2024-12-02 at 3 39 38 PM
> we already have the meeting URL in the reminder template, users can add the meeting URL, cancel the URL, etc if they want.

As per my understanding of the issue #17922 , the user expects the Reminder Email/SMS (made with default reminder template) triggered by workflow to have the meeting link. This PR adds Meeting URL in the default reminder template.

image

**Comment by Praashh** [comment]
Hey @kart1ka , thank for your contribution but we don't need meeting link by default atm, closing it for now

**Pull request closed without merging**